[macOS Monterey] Cannot find libpq-fe.h – how to disable PostgreSQL dependency

After installing all `brew` dependencies (`brew install autoconf automake bison freetype gd gettext icu4c krb5 libedit libiconv libjpeg libpng libxml2 libzip pkg-config re2c zlib`), I'm stuck at PostgreSQL check:

```
checking for PDO_DBLIB support via FreeTDS... no
checking for Firebird support for PDO... no
checking for MySQL support for PDO... yes
checking for MySQL UNIX socket location... no
checking for PDO includes... checking for PDO includes... /var/folders/x5/0v2t1jt94xx3k5n6v764v8qw0000gn/T/php-src-php-7.2.34/ext
checking Oracle OCI support for PDO... no
checking for ODBC v3 support for PDO... no
checking for PostgreSQL support for PDO... yes
checking for pg_config... not found
configure: error: Cannot find libpq-fe.h. Please specify correct PostgreSQL installation path
```

Is there a way to disable psql support by default?

_I'm on macOS Monterey, with no system PHP installed._

--- a/bin/install
+++ b/bin/install
@@ -168,7 +168,21 @@ construct_configure_options() {
   if [ "${PHP_WITHOUT_PDO_PGSQL:-no}" != "no" ]; then
     configure_options="$configure_options"
   else
-    configure_options="$configure_options --with-pdo-pgsql"
+    local operating_system=$(uname -a)
+
+    if [[ $operating_system =~ "Darwin" ]]; then
+      exit_if_homebrew_not_installed
+
+      local libpq_path=$(homebrew_package_path libpq)
+
+      if [ -n "$libpq_path" ]; then
+        configure_options="$configure_options --with-pdo-pgsql=$libpq_path"
+      else
+        export ASDF_PKG_MISSING="$ASDF_PKG_MISSING libpq"
+      fi
+    else
+      configure_options="$configure_options --with-pdo-pgsql"
+    fi
   fi
 
   echo "$configure_options"

